lessons/6_objects_intro/index.html

```diff
@@ -0,0 +1,332 @@
+        }
+
+        .illustration {
+            border: 0px
+        }
+
+        .reveal li {
+            margin-bottom: 0.5em;
+        }
+
+        .reveal pre {
+            width: 100%;
+        }
+
+        .reveal pre code {
+            cursor: pointer;
+        }
+
+        .reveal li pre {
+            width: fit-content;
+        }
+
+        .logo {
+            position: fixed;
+            /* Sit on top of the page content */
+            z-index: 2;
+            /* Specify a stack order in case you're using a different order for other elements */
+        }
+
+        .book {
+            background-color: #eee8d5;
+            text-align: left;
+            width: 100%;
+            display: flex;
+            min-height: 40vh;
+        }
+
+        .book img {
+            max-height: 35vh;
+        }
+
+        .book div {
+            padding: 5px;
+        }
+
+        .pages {
+            color: rgba(0, 0, 0, 0.5);
+        }
+
+        .price {
+            font-weight: bold !important;
+        }
+
+        .book-sample-img img {
+            max-height: 40vh;;
+        }
+    </style>
+</head>
+
+<body>
+<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo"/></a>
+<div class="reveal">
+    <div class="slides">
+        <section>
+            <h1>Lecture</h1>
+            <h3>Intro to objects</h3>
+        </section>
+
+        <section>
+            <h3>Let's say you want to sell books on your website...</h3>
+            <p class="fragment">You start with 1 book</p>
+            <div class="fragment">
+                <div class="book">
+                    <img src="zombie_raccoons.jpg">
+                    <div>
+                        <div>Name: <span>Zombie Raccoons & Killer Bunnies</span></div>
+                        <div>Price: <span class="price">20 &euro;</span></div>
+                        <div>Length: <span class="pages">643 pages</span></div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <h3>How your HTML might look like?</h3>
+            <div class="fragment">
+            <pre>
+					<code data-trim>
+<div class="book">
+	<div>
+    	<img class="image" src="zombie_raccoons.jpg">
+	</div>
+    <div>
+		<div>Name: <span>Zombie Raccoons & Killer Bunnies</span></div>
+		<div>Price: <span class="price">20 &euro;</span></div>
+		<div>Length: <span class="pages">643 pages</span></div>
+	</div>
+</div>
+                	</code>
+				</pre>
+            </div>
+        </section>
+
+        <section class="">
+            <h3>But all this information can come from elsewhere</h3>
+            <p class="fragment">Lest not hard-code it in HTML</p>
+            <div class="fragment">
+				<pre><code>
+let name = "Zombie Raccoons & Killer Bunnies";
+let image = "zombie_raccoons.jpg";
+let pageCount = 643;
+let price = 20;
+
+document.querySelector(".book img").src = image;
```

Comment:
you would need to explain querySelector, since we never used it.

Reply:
Good point, will do it during the class then. Guess we dont need separate slide for it, right?

Reply:
nope, just mention it and explain. Should be enough.

Reply:
added a link to mdn